internal/graph/models.graphqls

```diff
@@ -127,7 +127,12 @@ type Project {
   """
   Participants in this Project.
   """
-  participants: [Participant!]! @goField(forceResolver: true)
+  participants(offset: Int!, limit: Int!): [Participant!]! @goField(forceResolver: true)
```

Comment:
I don't know if offset and limit are required. Default offset should be 0, and limit 20. Just mention the defaults in the description.